src/components/DisplayArticle.jsx

```diff
@@ -1,18 +1,46 @@
 import { useParams } from "react-router-dom";
 import { getArticle } from "../utils/api";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { increaseVotes } from "../utils/api";
 
 const DisplayArticle = () => {
+  const clickHandler = (e, increment) => {
```

Comment:
you are invoking your function with an event parameter that you are not using. We can get rid of this if we are not using it in some way